docs/modules/ROOT/pages/how-tos/change-parameter.adoc

```diff
@@ -0,0 +1,37 @@
+= Change a Parameter
```

Comment:
According to our style guide we don't use Title Case for headings, cf. https://syn.tools/syn/references/style-guide.html#_asciidoc_style and https://docs.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings

```suggestion
= Change a parameter
```